libs/wallet-ui/src/components/navbar/index.tsx

```diff
@@ -1,9 +1,10 @@
 import type { ReactNode } from 'react'
-import { Settings } from '../icons/settings'
+// import { Settings } from '../icons/settings'
```

Comment:
**nitpick (non-blocking)**: Normally 🤨 , but it was explained in chat that this PR is WIP, but needs to be merged early due to some dependency complications. So 🤨 ✅.